sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md

```diff
@@ -104,11 +108,11 @@ To run the samples, ensure you have the following programs and tools installed o
     Run the following commands:
 
     ```bash
-    /$ cd azure-sdk-for-c/sdk/samples/iot/
+    ~$ cd azure-sdk-for-c/sdk/samples/iot/
 
-    /azure-sdk-for-c/sdk/samples/iot$ openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
-    /azure-sdk-for-c/sdk/samples/iot$ openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
-    /azure-sdk-for-c/sdk/samples/iot$ openssl x509 -noout -text -in device_ec_cert.pem
+    ~/azure-sdk-for-c/sdk/samples/iot$ openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
+    ~/azure-sdk-for-c/sdk/samples/iot$ openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
```

Comment:
I know this is a "pre-existing condition" - but is there any chance we could make the days timeout 30 instead fo 365?  We really want to force the issue that these are for test only and an earlier expiry puts that more in dev's face.

Also I'm not seeing the "Do NOT use these for production ever ever" warning we have other places.  Maybe the diff isn't showing, but if it's missing could we add please.

Reply:
Line 104 has the warning:

    **WARNING: Certificates created by these commands MUST NOT be used in production-level code on Windows or macOS.** These certificates expire after 365 days and are provided ONLY to help you easily understand CA Certificates. When productizing against CA Certificates, you will need to use your own security best practices for certificate creation and lifetime management.

Reply:
1 day instead of 30?

Reply:
Even better.